README.md

```diff
@@ -64,7 +87,12 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo) {
+ console.log(thingToDo);
+}
+
+doSomething("print this");
+    
 ```
```

Comment:
You've got a function `doSomething` defined and it does take an argument. When this function gets invoked it requires an argument that is a function. And inside the function definition the argument gets invoked.

```js
function doSomething(thingToDo) {
  thingToDo(); // call the function passed in, using parenthesis
}

function washTheDishes() {
  console.log("Inside the callback") 
}

doSomething(washTheDishes);
```